cfgov/regulations3k/jinja2/regulations3k/browse-regulation.html

```diff
@@ -41,44 +40,171 @@
+            <h1>{{ page.title }}</h1>
+            {% if version == regulation.effective_version %}
+                <div class="regulation-meta">
+                    <div class="a-date">
+                            Most recently amended {{ ap_date(version.effective_date) }}
+                    </div>
+                </div>
+            {% endif %}
+        </div>
+
         {% for block in page.header -%}
             {{ render_block.render(block, loop.index) }}
         {%- endfor %}
 
+        {% if version != regulation.effective_version %}
+        <div class="block block__flush-top">
+            <div class="m-full-width-text 
+                        m-notification
+                        m-notification__visible
+                        m-notification__warning">
+                {{ svg_icon('warning-round') }}
+                <div class="m-notification_content">
+                    <div class="h4 m-notification_message">This version is not current law.</div>
+                    <p class="m-notification_explanation">
+                        {% if version.effective_date < regulation.effective_version.effective_date %}
+                            You are viewing a previous version of this regulation with ammendments that went into effect on {{ ap_date(version.effective_date) }}.
+                        {% endif %}
+                        {% if version.effective_date > regulation.effective_version.effective_date %}
+                            You are viewing a future version of this regulation with ammendments that will go into effect on {{ ap_date(version.effective_date) }}.
+                        {% endif %}
+                    </p>
+                </div>
+            </div>
+        </div>
+        {% endif %}
+
+        <div class="block block__flush-top">
+                <ul class="m-list m-list__links m-full-width-text">
+                    {% if version == regulation.effective_version %}
+                    <li class="m-list_item">
+                        <a class="m-list_link" href="{{ routablepageurl(page, 'section', section_label=sections[0].label) }}">
+                            View current law
+                        </a>
+                    </li>
+                    {% else %}
+                    <li class="m-list_item">
+                        <a class="m-list_link" href="{{ routablepageurl(page, 'section', date_str=date_str, section_label=sections[0].label) }}">
+                            View this version
+                        </a>
+                    </li>
+                    {% endif %}
+
+                    {% if num_versions > 1 %}
+                    <li class="m-list_item">
+                        <a class="m-list_link" href="{{ routablepageurl(page, 'versions') }}">
+                            View all versions of this regulation
+                        </a>
+                    </li>
+                    {% endif %}
+
+                    <li class="m-list_item">
```

Comment:
The search link on the landing page is showing up for me on previous versions:

![search-link](https://user-images.githubusercontent.com/1862695/45375582-6acefb80-b5c3-11e8-9320-4d4a0c3d71bd.png)

I think that should only show up on the landing page of the current version.